plugins/emoticons.md

```diff
@@ -30,7 +30,7 @@ tinymce.init({
 
 ## Browser emoji support
 
-The emoticon plugin inserts unicode character codes, such as `\ud83d\ude03` for the smiley emoji. How emoji are rendered is dependent on the web browser and operating system of the user. As a result of this, some emoji may be rendered in black and white, or may not render.
+The emoticon plugin inserts unicode character codes by default, such as `\ud83d\ude03` for the smiley emoji. How emoji are rendered is dependent on the web browser and operating system of the user. As a result of this, some emoji may be rendered in black and white, or may not render.
```

Comment:
I debated about changing this, but didn't want to advertise the new image database too much. As such I just mentioned this applies to the plugin by default as this is what most users will run into.